code/obj/flock/cage.dm

```diff
@@ -202,6 +202,9 @@
 		else
 			if(hasvar(target, "health"))
 				var/absorption = min(absorb_per_process_tick, target:health)
+				if (ismob(target))
+					var/mob/M = target
+					M.TakeDamage(brute = absorption)
 				target:health -= absorption
```

Comment:
should this be in an else block? seems like we wouldn't want to do both

Reply:
huh, I have no idea why it isn't, good catch